docs/docs/guides/deploy/overview.mdx

```diff
@@ -17,3 +17,12 @@ By default, it runs a highly available ZITADEL instance along with a secure and
 - ZITADEL does not need many resources, 1 CPU and 512MB memory are more than enough. (With more CPU, the password hashing might be faster)
 - A cockroachDB or [🚧 Postgresql coming soon](https://github.com/zitadel/zitadel/pull/3998) as only needed storage
 - If you want to front ZITADEL with a reverse proxy, web application firewall or content delivery network, make sure to support [HTTP/2](../manage/self-hosted/http2)
+
+
+## Releases
+
+The easiest way to use ZITADEL is to run one of our container releases
```

Comment:
Why should a container be better then the rest?

Reply:
I didn't say "best" but easiest. To me containers are the fastest/most convenient way to get ZITADEL  up and running